scripts/docker-local-demo.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 (
-	docker ps >/dev/null && which docker-compose
+	docker --version && docker-compose --version
```

Comment:
I see. well issue was with docker-compose, which returns nothing if it's not there